prepare_pylarion.sh

```diff
@@ -1,14 +1,45 @@
-#!/bin/bash
+#!/bin/env bash
+
+echo "Clone Pylarion Library"
+
+if [[ -d pylarion ]]
+then
+    echo "Removing already existing dir $(pwd)/pylarion"
+    rm -rf pylarion
+fi
 
-echo "Clone Pylarion library"
-rm -rf pylarion
 git clone https://code.engineering.redhat.com/gerrit/pylarion
+if [[ ${?} -ne 0 ]]
+then
+    echo "Failed to clone 'pylarion' lib"
+    exit 1
+fi
 
-echo "Copy Pylarion config to home directory"
-rm -rf ~/.pylarion
-#cp rhos-qe-core-installer/tripleo/polarion/.pylarion ~/
-cp JUMP/.pylarion ~/
+# Config file is obtained separately
+if [[ ! -f .pylarion ]]
+then
+    echo ".pylarion config file not found"
+    exit 1
+fi
 
-echo "Setup Pylarion library"
-cd pylarion && python setup.py install && pip install --upgrade pip && pip install -r requirements.txt && cd ../
+# Do not copy .pylarion to user's $HOME unless requested
+if [[ -n ${1} ]]
```

Comment:
What is this argument? Any string will match. I don't see the value being used here.